src/main/java/com/nextstep/lotto/domain/Lottos.java

```diff
@@ -14,11 +14,7 @@ public int count() {
         return lottos.size();
     }
 
-    public LottoStatistics getStatistics(List<Integer> winningNumbers, int bonusNumber) {
-        if (winningNumbers == null || winningNumbers.isEmpty()) {
-            throw new IllegalArgumentException("winning numbers are not allowed empty value");
-        }
-        WinningLotto winningLotto = new WinningLotto(winningNumbers, bonusNumber);
+    public LottoStatistics getStatistics(WinningLotto winningLotto) {
```

Comment:
```suggestion
    public LottoStatistics createStatistics(WinningLotto winningLotto) {
```
로또 통계 객체를 생성하니 create이 적절해보이네요
create 대신에 도메인 용어로 좀더 읽기 좋은 단어를 찾아봐도 좋을 것 같아요

Reply:
의견 감사합니다 😄